lib/paperclip/storage/s3.rb

```diff
@@ -432,11 +403,7 @@ def flush_deletes #:nodoc:
         @queued_for_delete.each do |path|
           begin
             log("deleting #{path}")
-            if aws_v1?
-              s3_bucket.objects[path.sub(%r{\A/},'')]
-            else
-              s3_bucket.object(path.sub(%r{\A/},''))
-            end.delete
+            s3_bucket.object(path.sub(%r{\A/},'')).delete
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.